common/djangoapps/edxmako/tests.py

```diff
@@ -41,6 +41,29 @@ def test_marketing_link(self):
             link = marketing_link('ABOUT')
             self.assertEquals(link, expected_link)
 
+    # @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
```

Comment:
It looks like you had issues when running w/ DDT, but they went away when you split the test out?

I wonder if the decorators are conflicting..